services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/shared/guidelinesSection.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from "react";
```

Comment:
This file is mostly not new -- I'm not sure why it's diffing that way. It was moved from the `individualDataset` folder and I refactored the table to allow for rows to be selectable where it's used in the `TrialRunner`.